packages/wasm/src/index.ts

```diff
@@ -1,34 +1,86 @@
-import { readFile } from 'fs';
-import { resolve } from 'path';
+import * as fs from 'fs';
+import * as path from 'path';
+import { promisify } from 'util';
+import { createHash } from 'crypto';
 
 import { Plugin } from 'rollup';
 
 import { RollupWasmOptions } from '../types';
 
+const fsStatPromise = promisify(fs.stat);
+const fsReadFilePromise = promisify(fs.readFile);
```

Comment:
It looks like the fsPromises API for both of these has been around since at least Node 10.x (the minimum engine for plugin-wasm), unless there's a reason they can't be used?

Reply:
Oh, sure I'll gladly fix that. This was based on code in the URL plugin:

https://github.com/rollup/plugins/blob/55a9cb20d7686750aa104d6a45a7d6834f57ba4a/packages/url/src/index.js#L10-L11

So I didn't think much of it and thought it was a backwards-compatibility issue.

Reply:
Fixed now.

Reply:
Thanks for calling that out, definitely something to keep in mind for cleanup!